proto/trtl/v1/trtl.proto

```diff
@@ -94,8 +94,9 @@ message BatchReply {
 
 message CursorRequest {
     bytes prefix = 1;       // the prefix to range over, if nil all objects are returned
-    string namespace = 2;
-    Options options = 3;
+    bytes seek_key = 2;     // a key to seekto within the prefix (optional)
+    string namespace = 3;
+    Options options = 4;
```

Comment:
The idea behind `seek_key` is that the user wants to move forward inside of a common prefix range. For example, let's say the keys are:

- aa1
- aa2
- aa3
- aa4
- aa5
- b1
- b1
- b3

If the user only cares about keys that start with 'a', but wants to start iterating from a3, they would make the request as follows:

```go
&CursorRequest{Prefix: []byte('a'), SeekKey: []byte('a3')}
```